de/news/_posts/2017-04-14-deutsche-ruby-mailingliste.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: news_post
+title: "Deutschsprachiges Ruby-Forum schließt und wird durch Mailingliste ersetzt"
+author: "Marvin Gülker"
+translator:
+date: 2017-04-14 00:00:00 +0000
+lang: de
+---
+
+Das deutschsprachige Ruby-Forum schließt und wird durch eine
+Mailingliste ersetzt.
+
+Das [einzige deutschsprachige Forum zur Programmiersprache Ruby][1]
+befand sich [seit ca. 2014][2] nicht mehr in aktiver Wartung. Bis
+heute war es nicht möglich, jemanden zu finden, der die Administration
+übernehmen wollte, weshalb die Moderation und der bisherige Admin
+nunmehr die Konsequenzen gezogen haben und [das Forum schließen
+werden][3].
+
+Ab etwa Anfang Juni wird das Forum — und die dazugehörige Wiki — nach
```

Comment:
Ich habe das Wort oft genug im Femininum in Gebrauch gesehen. Ich habe bisher immer geglaubt, dass das mit dem Wortursprung zusammenhinge und die Neutrum-Version nur eingedeutscht ist (so wie man ja heute auch „Büro“ schreibt und nicht mehr „Bureaux“ [hoffentlich war das richtig in Französisch]). Der Duden gibt ja leider keine Begründung für seine Schreibweise... Was den Duden angeht, stehe ich teilweise auch der neuen Rechtschreibung eher ablehnend gegenüber (ich werde privat niemals „Fantasie“ schreiben, das ist und bleibt „Phantasie“ für mich). Es ist mir bis heute unbegreiflich, weshalb Deutschland die Rechtschreibung per Gesetz regeln wollte, was für ein Unfug.

Naja, für ruby-lang.org sollte die Duden-Schreibung aber wohl ausschlaggebend sein. Ich passe das entsprechend an.